docs/rcait.md

```diff
@@ -15,49 +15,89 @@ component and how PerformanceAnalyzer interacts with ElasticSearch.
     
-`__@AMetric__` 
-
-This helps us specify the metrics that will be pured over the RCA graph. It has multiple sub
--fields.
-- _name_ : The metric we are filling in. The expected parameter is one of the metrics classes
+__@AMetric__ 
+This helps us specify the metrics that will be pured over the RCA graph. It has multiple sub-fields.
+
+Let's see an example:
+```java
+@AMetric(name = CPU_Utilization.class,
+    dimensionNames = {SHARDID_VALUE, INDEX_NAME_VALUE, OPERATION_VALUE, SHARD_ROLE_VALUE},
+    tables = {
+        @ATable(hostTag = HostTag.DATA_0,
+            tuple = {
+                @ATuple(dimensionValues = {"0", "logs", "bulk", "p"},
+                    sum = 0.0, avg = 0.0, min = 0.0, max = 0.0),
+                @ATuple(dimensionValues = {"1", "logs", "bulk", "r"},
+                    sum = 0.0, avg = 0.0, min = 0.0, max = 80.0),
+                @ATuple(dimensionValues = {"2", "logs", "bulk", "p"},
+                    sum = 0.0, avg = 0.0, min = 0.0, max = 10.0)
+            }
+        ),
+        @ATable(hostTag = HostTag.DATA_1,
+            tuple = {
+                @ATuple(dimensionValues = {"0", "logs", "bulk", "r"},
+                    sum = 0.0, avg = 0.0, min = 0.0, max = 50.0),
+                @ATuple(dimensionValues = {"1", "logs", "bulk", "p"},
+                    sum = 0.0, avg = 0.0, min = 0.0, max = 5.0),
+                @ATuple(dimensionValues = {"2", "logs", "bulk", "r"},
+                    sum = 0.0, avg = 0.0, min = 0.0, max = 11.0)
+            }
+        )
+    }
+)
+```
+
+- _name_ : The Metric graph node that will be emitting these values. The expected parameter is one of the metrics classes
  in `com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.metrics`. The
  metrics class that you specify, should have a `static final` field called `NAME` (`CPU_Utilization`)
- and that will be used to determine the metric table.
+ and that will be used to determine the name of the metric table.
 - _dimensionNames_ : For the dimension names for a metric, please refer to the docs 
-[here](https://opendistro.github.io/for-elasticsearch-docs/docs/pa/reference/).
-- _tables_ : This specifies a table for the metric. The table should be a 5 second snapshot
+[here](https://opendistro.github.io/for-elasticsearch-docs/docs/pa/reference/). Each of the dimension
+will be a column in the rca.sqlite table.
+- _tables_ : This specifies one or more table(s). The table should be a 5 second snapshot
  of the metrics, similar to what exists in metricsdb files. The table is an array type
  , therfore it gives you the flexibility of specifying a different metrics table for
- different nodes in the cluster. This can be used to push different metrics to the node that
+ each node or a group of nodes in the cluster. In the above example, the table one is to be emitted by
+ `datanode0` and table 2 by `datanode1`. This can be used to push different metrics to the node that
```

Comment:
Should this be `DATA_0` and `DATA_1` instead of `datanode0` and `datanode1` resp ?